docs/getting-started/sync-with-github.md

```diff
@@ -75,9 +75,9 @@ When synchronizing with GitHub using the Free Plan, the following occurs:
 
 When synchronizing with GitHub using the Pro or Enterprise Plan, the following occurs: 
 
-- When you first create a build in Amplication Pro and Enterprise Plans, it will create a branch named Amplication and create inside. Pull Request inside a branch that is created named “Amplication.”
-- Subsequent builds will create additional commits within the Amplication branch if there is an open pull request. If there is no open pull request, it will open a new pull request.
-- After the PR has been merged, the next build will create a new Pull Request from the first commit of the default branch. Subsequent builds will add additional commits until the next merge, and so on.
+- **Initial Build Creation**: When you create your first build in Amplication, a new branch labeled `amplication` is created. All Pull Requests from Amplication are opened against this branch
```

Comment:
This is only valid for smart git, right?

Reply:
yes. we are not creating this branch in the basic strategy